amundsen_application/static/js/components/DashboardPage/QueryListItem/CodeBlock.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { CopyBlock, atomOneLight } from 'react-code-blocks';
+
+const LANGUAGE = "sql";
+
+const CodeBlock = ({text}) => {
+  return (
+    <CopyBlock
+      text={text}
+      language={LANGUAGE}
+      theme={atomOneLight}
+      showLineNumbers={false}
+      wrapLines={true}
+      codeBlock={true}
+    />
+  );
+}
+
+export default CodeBlock;
```

Comment:
This module is by itself so we can lazy load it, as the dependencies of CopyBlock are numerous and heavy.